runtime/vm/guardedstorage.c

```diff
@@ -48,6 +49,15 @@ j9gs_initializeThread(J9VMThread *vmThread)
 	if (IS_THREAD_GS_INITIALIZED(vmThread)) {
 		success = 1;
 	} else {
+
+		int32_t supportsGuardedStorageFacility = 0;
```

Comment:
Would it better to have supportsGuardedStorageFacility type BOOLEAN? Here and below